webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report_list_reports.js

```diff
@@ -66,10 +66,6 @@ var Reports = (function() {
        activate: function(url) {
            url = url || null;
            if (loaded) return;
-           if (Columns.value() === null) {
-               // the storage hasn't returned yet!
-               return Columns.load(Reports.activate);
-           }
```

Comment:
I do not understand why you remove this code, and I guess it's what causes the error I'm seeing.